react/features/invite/components/DialInNumbersForm.js

```diff
@@ -408,7 +398,7 @@ class DialInNumbersForm extends Component {
         const numbers = this._formatNumbers(dialInNumbers);
 
         this.setState({
-            selectedNumber: numbers[0]
+            selectedNumber: numbers[0].props.children
```

Comment:
I'm not sure how I feel about a parent reaching down into a child for state that it can have access to itself. What are the alternatives here?

Reply:
The more I think about it the more I think maybe it's okay. I'm not used to it and there is access to the number dialInNumbers, which tells me maybe something with how the default is set needs to change. I'd still be curious about alternatives, though.